manager/src/main/resources/application.properties

```diff
@@ -41,7 +41,7 @@ event-bridge.rhoas.sso.red-hat.refresh-token=empty
 %dev.quarkus.hibernate-orm.log.bind-parameters=true
 
 # SSO
-%dev.quarkus.oidc.auth-server-url=http://localhost:8180/auth/realms/event-bridge-fm
+%dev.quarkus.oidc.auth-server-url=http://192.168.49.2:30007/auth/realms/event-bridge-fm
```

Comment:
@robpblake @danielezonca @lucamolteni @kostola @sutaakar @Srihari1192 
FYI please note that I had to move keycloak from `docker-compose` to `minikube` because of the new authentication flow of the ingress application. Usually minikube runs on `192.168.49.2`, but unfortunately minikube does not support assigning a static IP to the cluster (https://github.com/kubernetes/minikube/issues/951) so this might not be always true (for example if you stop and restart the cluster). There might be some tricks we can try, but atm I'd set it to this default value and look at it as soon as I have time to investigate.
The instructions for the dev env have been updated accordingly.